editor/src/views/Crashes/SwapAddressButton.js

```diff
@@ -3,12 +3,14 @@ import { Button } from "reactstrap";
 import ConfirmModal from "../../Components/ConfirmModal";
 import { UPDATE_CRASH } from "../../queries/crashes";
 
-const SwapAddressButton = ({ data, crashRefetch, ...props }) => {
+const SwapAddressButton = ({ data, crashRefetch, setEditField, ...props }) => {
   const [isConfirmModalOpen, setIsConfirmModalOpen] = useState(false);
 
   const crash = data?.crash;
 
   const toggleModal = () => {
+    // make sure we are not in edit mode on a field
+    setEditField("");
```

Comment:
so that if we are in edit mode on a field and click the swap address button, we get out of edit mode